ext/date/php_date.c

```diff
@@ -841,6 +841,11 @@ static zend_string *date_format(const char *format, size_t format_len, timelib_t
 	return string.s;
 }
 
+PHPAPI zend_string *php_format_date_ex(const char *format, size_t format_len, timelib_time *t, bool localtime)
```

Comment:
I think I would prefer it if `timelib_time` does not get (more) exposed to the external API. Would it perhaps make more sense to have this function accept a `php_date_obj` instead of a `timelib_time`?

Reply:
Agreed. I vaguely recall having talked about exactly that in room 11 actually.
I changed the signature and function name.